manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/impl/WeWorkRobotAlertNotifyHandlerImpl.java

```diff
@@ -71,6 +84,34 @@ public void send(NoticeReceiver receiver, NoticeTemplate noticeTemplate, Alert a
         }
     }
 
+    private WeWorkWebHookDto checkNeedAtNominator(NoticeReceiver receiver, Alert alert) {
+        if (StringUtils.isBlank(receiver.getPhone()) && StringUtils.isBlank(receiver.getTgUserId())) {
+            return null;
+        }
+        WeWorkWebHookDto weWorkWebHookTextDto = new WeWorkWebHookDto();
+        weWorkWebHookTextDto.setMsgtype(WeWorkWebHookDto.TEXT);
+        WeWorkWebHookDto.TextDTO textDto = new WeWorkWebHookDto.TextDTO();
+        String alertMessage = String.format("警告对象：%s\n详情：%s", alert.getTarget(), alert.getContent());
```

Comment:
Can these be replaced with `i18n`?

Reply:
At present, we do not know the back-end i8n, and we can modify it after learning